apps/glusterfs/brick_entry.go

```diff
@@ -12,6 +12,7 @@ package glusterfs
 import (
 	"bytes"
 	"encoding/gob"
+
```

Comment:
This is recommended practice. One empty line before referencing non-standard packages.